src/components/paragraph/_index.scss

```diff
@@ -3,13 +3,13 @@
 .c-paragraph {
 	--paragraph-truncation: var(--c-paragraph--webkit-line-clamp, "none");
 
-	overflow: hidden;
-	display: -webkit-box;
-	-webkit-box-orient: vertical;
-	-moz-box-oriented: vertical;
-	text-overflow: ellipsis;
-
 	@include scss.component-properties("paragraph");
 
+	overflow: var(--paragraph-overflow, hidden);
```

Comment:
Moving these below to make use of the default value of the `var` to avoid these properties not being there as intended when being overridden with blocks